frontend/src/components/landingPage/MenuStepTwo.vue

```diff
@@ -11,7 +11,7 @@
 					<option>=</option>
 				</select>
 				<input class="form-control" id="valueInput" type="text" placeholder="Input Value" v-model="selectedValue" @input="sendValue">  
-				<h6 id="nanInformation" v-show="notANumber">Value should be numerical</h6>           
+				<h6 id="nanInformation" v-show="notANumber">Number Please</h6>           

```

Comment:
This is a regression. We fixed this in last weeks PR already. Please make sure you merge back these changes when developing in parallel.

Reply:
This has been fixed and commit has been pushed.